regulation-worker/cmd/main.go

```diff
@@ -45,8 +46,12 @@ func main() {
 }
 
 func Run(ctx context.Context) {
+	admin.Init()
```

Comment:
question: why do we need this here now?

Reply:
Because the backend config needs to register an admin handler, see [here](https://github.com/rudderlabs/rudder-server/blob/35238ebf0d426d0e21a7fb0e827e603f95180a66/config/backend-config/backend-config.go#L247).

Reply:
Maybe we can:
1. Move this registration outside and only do this register for rudder-server
2. Or ensure that registration can happen without admin.Init() - some kind of self init way

Reply:
Makes sense, although I would prefer to move it once we need the backend config to function without the admin package. I'm saying this because I have done lots of changes here already. The more I keep doing the more likely this PR could break something.